From a4a7a98f01c430ecdaf4e2dbee3c94d6a9ee4ed1 Mon Sep 17 00:00:00 2001
From: Tobias <ruck.tobias@gmail.com>
Date: Mon, 27 Jun 2022 01:39:54 +0200
Subject: [PATCH 165/196] Set min testnet difficulty to 1 (#258)

Currently, the PoW limit for the testnet is needlessly high with a minimum difficulty of ~16. This change reduces the difficulty to 1.
---
 src/chainparams.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/chainparams.cpp b/src/chainparams.cpp
index ecd5775a8..90e387d0b 100644
--- a/src/chainparams.cpp
+++ b/src/chainparams.cpp
@@ -205,7 +205,7 @@ public:
         consensus.nSubsidyHalvingInterval = 210000;
 
         consensus.powLimit = uint256S(
-            "0000000010000000000000000000000000000000000000000000000000000000");
+            "00000000ffffffffffffffffffffffffffffffffffffffffffffffffffffffff");
         // two weeks
         consensus.nPowTargetTimespan = 14 * 24 * 60 * 60;
         consensus.nPowTargetSpacing = 2 * 60;
-- 
2.39.5 (Apple Git-154)

